### PR TITLE
Word Search problem completed

### PR DESCRIPTION
**Pull request by Shubh-Krishna**
# Word Search Completed

image

### CODE REVIEW

**Comment by Shubh-Krishna** [comment]
Issue : #19

File: Others/Word_Search/Word_Search.cpp
Comment (bh-g) [inline Others/Word_Search/Word_Search.cpp:1]: Rename this file to solution.cpp.

File: Others/Word_Search/Word_Search.cpp [hunk 1]
```diff
@@ -0,0 +1,73 @@
+#include <bits/stdc++.h>
+using namespace std;
+
```
Comment (bh-g) [inline Others/Word_Search/Word_Search.cpp:3]: Delete these lines.

File: Others/Word_Search/Word_Search.cpp [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    {
+        cout << "True" << endl;
+    }
+    else
+    {
+        cout << "False" << endl;
+    }
+
+    return 0;
+}
```
Comment (bh-g) [inline Others/Word_Search/Word_Search.cpp:73]: Delete this part.

**Review by bh-g — CHANGES_REQUESTED** [review]
Rest is fine.

**Comment by Shubh-Krishna** [comment]
I have changed the code according to the review.
If there is any other issue please notify 😊😊
Thank you for reviewing😊😊

**Pull request merged**